spec/features/admin/widgets/cards_spec.rb

```diff
@@ -135,4 +135,115 @@ def attach_image_to_card
       make_visible: true)
     expect(page).to have_field('widget_card_image_attributes_title', with: "clippy.jpg")
   end
-end
\ No newline at end of file
+
+  context "Translations" do
```

Comment:
I've seen we've got specs using `it_behaves_like` and `shared_examples` methods. Do you think it makes sense to do something similar with `translatable` specs?

Sorry I didn't comment about it earlier, since it could have been included in #1613; I only saw the duplication now that I was reviewing this PR.

Reply:
Yes, there is quite a bit of test duplication across resources :/. @voodoorai2000 said to leave it that way for now and worry about refactoring to shared examples later. I might try to do it anyway though if I have time.